stdlib/3/subprocess.pyi

```diff
@@ -151,7 +151,29 @@ else:
                    start_new_session: bool = ...,
                    pass_fds: Any = ...) -> int: ...
 
-if sys.version_info >= (3, 4):
+if sys.version_info >= (3, 6):
+    # 3.6 added encoding
+    def check_output(args: _CMD,
+                     bufsize: int = ...,
+                     executable: _TXT = ...,
+                     stdin: _FILE = ...,
+                     stderr: _FILE = ...,
+                     preexec_fn: Callable[[], Any] = ...,
+                     close_fds: bool = ...,
+                     shell: bool = ...,
+                     encoding: str = ...,
```

Comment:
Thanks for reviewing!

- str is changed to Optional[str]

- errors parameter is added for check_output() function

- new parameters are moved to the end and '*' is added

- According to document only subprocess.call(), subprocess.check_output() and Popen.__init__() are using `encoding` and `errors`. The two parameters are already there